### ISSUE TITLE
Console i/o

### ISSUE THREAD

**Issue by Mellvik** [ghaerr/elks#124]
When using BIOS console I/O, the system softhangs on input of a char < (ascii)6 (such as ctrl-C). The problem goes away when the **cmp al,#6** instruction  in kbd_timer() is removed. (bioscon.c)

**Comment by Mellvik** [comment]
Major improvement!! Virtual consoles now work, but: text written to an invisible console, does not show. The cursor is in the right position, that's it (tested on physical 286). Fuctions keys otherwise not tested. BTW (possibly unrelated, and has existed for a while): ^Z hangs the system. Maybe all the job control stuff should be eliminated?

Note: feedback after #126.

**Comment by mfld-fr** [comment]
This issue deals with several mixed points that would be addressed one by one:
- [x] Control-C (or any char < 6) in BIOS console hangs the system : not reproduced
- [x] Control-Z in BIOS console hangs the system : not reproduced
- [x] Text written to hidden virtual console is not visible: confirmed - see below

**Comment by mfld-fr** [comment]
Just tested on latest master with a BIOS console: Control-C and Control-Z do not hang the system any more. But Control-C in `ash` still causes the spurious display of `HOME not set` (already tracked by #125).

**Comment by mfld-fr** [comment]
Problem confirmed for virtual consoles and BIOS driver under QEMU. After enabling `getty` for slots 2 and 3 in `/etc/inittab`, cursor is displayed but no text visible when switching to VC3, and no cursor neither text visible at all in VC2. Can login in VC3, but not in VC2.

**Comment by mfld-fr** [comment]
Found the cause of the VC2 problem: in `qemu.sh` script, the requested VGA card is the Cirrus one (`-vga cirrus`), but the VGA BIOS is not more consistent since QEMU 2.2. Solved that problem with option `-vga std`. Now having the same behaviour as reported by @Mellvik.

**Comment by mfld-fr** [comment]
Hidden text problem fixed in #216.
